websites/ribbons/public/css/index.css

```
@@ -523,6 +570,239 @@ footer a:hover {
+}
+
+.site-nav-theme:after {
+  position: absolute;
+  top: 0;
+  left: 0;
+  content: '';
+  width: var(--size);
+  height: var(--size);
+  background-repeat: no-repeat;
+  background-position: center;
+  background: var(--theme-switch);
```

Stylelint (17.12.0)

[error] Overridden property "background-position" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)